src/AWS.Deploy.Recipes.CDK.Common/RecipeProps.cs

```diff
@@ -70,8 +71,9 @@ public interface IRecipeProps<T>
     /// <summary>
     /// A representation of the settings transferred from the AWS .NET deployment tool to the CDK project.
     /// </summary>
-    /// <typeparam name="T"></typeparam>
-    public class RecipeProps<T> : IRecipeProps<T>
```

Comment:
These are breaking changes here removing `ECRRepositoryName` and adding a new `DeploymentBundleSetting` expecting the CDK project to understand. You got around this by modifying all of our recipes but if a user has a deployment project then this change would break them when they take an updated version of the tooling. We need to find a way to implement this feature with a breaking change.

Reply:
I added the `DeploymentBundleSettings` in the `RecipeProps` as a `string` and reverted all modifications to the CDKTemplates. These way all persisted CDK projects will still work with the updated version of the tooling.